yb-voyager/cmd/exportData.go

```diff
@@ -346,14 +346,14 @@ func addLeafPartitionsInTableList(tableList []sqlname.NameTuple) (map[string]str
 		allLeafPartitions := GetAllLeafPartitions(table)
 		switch true {
 		case len(allLeafPartitions) == 0 && rootTable != table: //leaf partition
-			partitionsToRootTableMap[qualifiedCatalogName] = rootTable.ForKey() // Unquoted->MinQuoted map as debezium uses Unquoted table name
+			partitionsToRootTableMap[qualifiedCatalogName] = rootTable.AsQualifiedCatalogName() // Unquoted->Unquoted map as debezium uses Unquoted table name
```

Comment:
Let's try to use utils.StructMap wherever we want to use table as a key. 
Using map[string].. is fine but quite error prone. 
If there are any concerns in this case, given that we manually create the name tuples for partitions, let's discuss.

Reply:
in this case, it could be `utils.StructMap[NameTuple, NameTuple]`

Reply:
This map is just used to put it in MSR or dbzm's config and no where else. So anyway then I had to iterate to get string->string map at two places. So I think it should be fine to get it like this itself from here?